guides/theme-quality-assurance.md

```diff
@@ -0,0 +1,355 @@
+* Social links and features should be associated with the brand’s social channels and handles, e.g. should mention brand social handle, correct social URL, etc.
+* Theme product detail views should be consistent across all products and should not have to be customized product-by-product
+
+#### Product Reviews
+{:.no_toc}
+
+* Product detail views should have review stars and quotes
+* Product detail views should only show reviews for the product featured on the current view
+
+### Cart
+{:.no_toc}
+
+* Cart view should be width-responsive and should behave as expected at phone (landscape and portrait), tablet (landscape and portrait), and desktop resolutions and aspect ratios
+* Cart view should display all items added to cart
+* When cart item quantity is changed, the update button should behave as expected
+* The remove item button should function properly
+* Active promotions should be displayed in the cart and the cart total should show their effect
+* When a promotion is active in the cart, the promotion name and discount should be displayed
+* When a promotion is unable to be used, an error message should explain why
+* Cart subtotal should display the correct subtotal amount
+* When a cart modal is part of the theme, the cart modal should display properly and fail gracefully in target browsers
+* When a cart modal is part of the theme, the cart modal should allow the user to modify items in their cart
+* When the checkout button is tapped/clicked, the user is directed to the first incomplete checkout step
+* When a user deletes all items from their cart, the cart should display no items
+* When a user attempts to proceed to checkout with an empty cart, the cart should display an error message and should not redirect to checkout
+* Tracking scripts should track the cart view correctly
+* Tracking scripts should track an item removed from the cart correctly
+* Tracking scripts should track a discount added to the cart correctly
+* The checkout call-to-action button should be prominent and should be visible without scrolling
+* Cart tap/click targets should be visible, large enough to be tapped/clicked, and unobscured by the edges of the viewport window
+
+### Shopper-Users
+{:.no_toc}
+
+* Shopper-user should be able to log in
+* Shopper-user should be able to sign up
+* Shopper-user should be able to access account
+* Shopper-user should be able to log out
+* Shopper-user should be able to edit information, email, and password
+* When a Shopper-user performs an unsuccessful sign up, log in, or account information submit attempt, they should be presented with error messages that are clear, concise, and accessible to a 5th-grade reading level
+* When a Shopper-user performs an successful sign up, log in, log out, or account information submit attempt, they should be presented with a clear success message
+* A guest shopper-user should be able create an account with a password via an emailed link after checkout
```

Comment:
"Shopper-user" is capitalized on lines 186, 187, and 189, but it is not capitalized on lines 188 and 190--for consistency purposes, should we choose to capitalize or lowercase all instances of "shopper-user"?

Reply:
Oh! Also a very good catch on your end. 👍